docs/source/configuration/authorization.mdx

```diff
@@ -417,19 +417,18 @@ If _every_ requested field requires authentication and a request is unauthentica
 ### `@policy`
```

Comment:
Is it too late to change the name? 😅 Seems like `requiresPolicies` would be more consistent with the other `requiresScopes` directive.